assets/eslint.config.mjs

```diff
@@ -0,0 +1,85 @@
+import { fixupConfigRules, fixupPluginRules } from "@eslint/compat";
+import typescriptEslint from "@typescript-eslint/eslint-plugin";
+import react from "eslint-plugin-react";
+import jsxA11Y from "eslint-plugin-jsx-a11y";
+import tsParser from "@typescript-eslint/parser";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import js from "@eslint/js";
+import { FlatCompat } from "@eslint/eslintrc";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const compat = new FlatCompat({
+    baseDirectory: __dirname,
```

Comment:
This file seems to be 4-spaced, rather than our usual 2-spaced.

Reply:
Hmm yeah thank you. I guess the devs of the migrator tool are 4-spacers 😠